src/main/java/ip/Ui.java

```diff
@@ -0,0 +1,59 @@
+package ip;
+
+import java.util.Scanner;
+
+/**
+ * Handles all communications with the user.
+ *
+ * @author Jonathan Lam
+ */
+public class Ui {
+    private Scanner input = new Scanner(System.in);
+    private final String greeting = "  ___          ___  \n"
```

Comment:
Following the 2103T code standards, greeting should be all caps since its a constant.